### DISCUSSION TITLE
Integrate a Chatbot developed with Promptflow into MS Copilot Studio and MS Teams

### DISCUSSION THREAD

**Discussion by HyeonjeCho** [microsoft/promptflow#2322] [Q&A] [answered]
I'm interested in a chatbot on my documents developed with Promptflow into MS Copliot Studio (copilotstudio.microsoft.com) and ultimately deploying it for use in MS Teams. 
I'm seeking guidance on the process of deploying Promptflow through Copilot Studio or integrating it directly with MS Teams. 
Any insights or instructions on this integration would be greatly appreciated.
Thanks

**Comment by brynn-code** [comment, marked as answer]
Hi @HyeonjeCho , we don't have a full path guidance of your requirements, but a general approach to achieve your goal can be simply summarized with 3 steps:
1. Deploy your flow as an endpoint, for example using Azure app service or deploy as a managed online endpont. Here are the related docs: [app service](https://microsoft.github.io/promptflow/how-to-guides/deploy-a-flow/index.html), [managed online endpoint](https://learn.microsoft.com/en-us/azure/machine-learning/prompt-flow/how-to-deploy-for-real-time-inference?view=azureml-api-2)
2. Consume the endpoint from copilot studio.
3. Follow-up steps, integrate with teams or something else. (Sorry that I have little knowledge about this, but there should be common docs.)
